src/Domain/ValueObjects/ShoppingItemFilterEnum.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Lindyhopchris\ShoppingList\Domain\ValueObjects;
+
+class ShoppingItemFilterEnum
+{
+    /**
+     * @var string
+     */
+    private string $all;
+
+    /**
+     * @param string $all
+     */
+    public function __construct(string $all)
```

Comment:
Required: rename the constructor parameter here to match whatever name you change the class property to.